tests/test_planning.py

```diff
@@ -245,23 +245,33 @@ def test_planning_determinism():
         cry_option, [robot_var],
         lambda s, g, rng, objs: pos_params_space.sample())
 
-    def _simulator(s, a):
-        ns = s.copy()
-        if a.arr.item() < -1:
-            ns[robby][0] = 1
-        elif a.arr.item() < 0:
-            ns[robin][0] = 1
-        elif a.arr.item() < 1:
-            ns[robin][1] = 1
-        else:
-            ns[robby][1] = 1
-        return ns
-
     goal = {asleep([robby]), asleep([robin]), cried([robby]), cried([robin])}
     task1 = Task(State({robby: [0, 0], robin: [0, 0]}), goal)
     task2 = Task(State({robin: [0, 0], robby: [0, 0]}), goal)
-    option_model = create_option_model("oracle")
-    option_model._simulator = _simulator  # pylint:disable=protected-access
+
+    class _MockEnv(BaseEnv):
```

Comment:
@ronuchit if you don't like this pattern, we could also create one BaseMockEnv somewhere that is just BaseEnv with all of the abstractmethods not abstract, and then inherit. but I'm not sure where that would live.